### ISSUE TITLE
Stable Diffusion 2.0 released

### ISSUE THREAD

**Issue by AlfredJKwack** [joelparkerhenderson/stable-diffusion-macos-install-help#8]
Hi,

Can you confirm whether this guide will support Stable Diffusion 2.0 which was just released?

**Comment by joelparkerhenderson** [comment]
Yes. Stable Diffusion 2.0 feature request is in the queue, waiting on this: https://github.com/invoke-ai/InvokeAI/issues/1545

**Comment by AlfredJKwack** [comment]
Thanks @joelparkerhenderson !
